devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -161,18 +228,22 @@ def on() {
 
 def off() {
 	log.debug "sending off"
-	[
-		zwave.basicV1.basicSet(value: 0x00).format(),
-		zwave.basicV1.basicGet().format()
-	]
+	if (isYale()) {
+		[
+			secure(zwave.basicV1.basicSet(value: 0x00)),
+			secure(zwave.basicV1.basicGet()),
+			secure(zwave.alarmV2.alarmGet(zwaveAlarmType: 0x07))
+		]
+	} else {
+		[
+			zwave.basicV1.basicSet(value: 0x00).format(),
+			zwave.basicV1.basicGet().format()
+		]
+	}
```

Comment:
I'd structure this to just append the `alarmGet` for Yale devices instead of all the duplication.